src/main/java/Control/Scenario/UC_Auth.java

```diff
@@ -209,24 +204,48 @@ public UC_Auth(@NotNull DbProvider dbProvider) {
             T_AccessPrivilegeJournal t_accessPrivilegeJournal = I_AccessPrivilegeJournal.retrieveValidForUser(db.getConn(), db.getPs(), db.getRs(), authDb.getUser().getUserID());
             authDb.setIsadmin(t_accessPrivilegeJournal.getA_AccessPrivilegeID() == T_AccessPrivilegeJournal.ACCESS_PRIVILEGE_ID_ADMIN);
 
-            db.afterSqlExecution(true);
+            db.afterOkSqlExecution();
 
             jsonResponse.setStatus(HttpServletResponse.SC_OK);
             jsonResponse.setMessage("Login successful.");
             jsonResponse.setData(authDb);
         } catch (AuthenticationException e) {
-            db.afterSqlExecution(false);
+            db.afterExceptionInSqlExecution(e);
 
             jsonResponse.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
         } catch (SQLException e) {
-            db.afterSqlExecution(false);
+            db.afterExceptionInSqlExecution(e);
 
             jsonResponse.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
             jsonResponse.setMessage("Internal server error.");
         }
         return jsonResponse;
     }
 
+    public void LogLoginIntoDatabase(int userId, HttpServletRequest request) {
```

Comment:
1. na  tejto vrstve uz nie request, ale cisto, parametre/pripadne urobit modelovy objekt
2. anglicky sa povie "log to" nie log into

Reply:
aha sry haha je to taky slovny mindfuck log login to database a log login into database znamenaju troska dve rozne (ak to vidis :) ) veci, ale zrejme to mas ty spravne, dobre neries

Reply:
Jaj takto si to myslel :D Aj tak je to ndolezita funkcionalita ale dik za povsimnutie